models/src_facebook_ads.yml

```diff
@@ -242,3 +250,127 @@ sources:
             description: Link of the template app for android
           - name: template_app_link_spec_iphone
             description: Link of the template app for iphone
+
+      - name: demographics_country
+        identifier: '{{ var("facebook_ads_demographics_country_identifier", "demographics_country") }}'
+        description: '{{ doc("demographics_country") }}'
+        config:
+          enabled: "{{ var('ad_reporting__facebook_ads_enabled', true) and var('facebook_ads__using_demographics_country', false) }}"
+        columns:
+          - name: account_id
+            description: '{{ doc("account_id") }}'
+          - name: date
+            description: '{{ doc("date") }}'
+          - name: _fivetran_id
+            description: '{{ doc("country_id") }}'
+          - name: country
+            description: '{{ doc("country") }}'
+          - name: impressions
+            description: '{{ doc("impressions") }}'
+          - name: inline_link_clicks
+            description: '{{ doc("inline_link_clicks") }}'
+          - name: reach
+            description: '{{ doc("reach") }}'
+          - name: cost_per_inline_link_click
+            description: '{{ doc("cost_per_inline_link_click") }}'
+          - name: cpc
+            description: '{{ doc("cpc") }}'
+          - name: cpm
+            description: '{{ doc("cpm") }}'
+          - name: ctr
+            description: '{{ doc("ctr") }}'
```

Comment:
I don't see these fields in the macro, should these be removed?

Reply:
They're not, but I thought it was easy enough to document them for users in case they want to pass them in. I prefer to leave them in unless you have a strong opinion otherwise